Snakefile-sorting-hat

```diff
@@ -131,15 +131,18 @@ rule get_experiment_metadata:
         source {workflow.basedir}/bin/metadata_retrieval_functions.sh
 
         perl {workflow.basedir}/bin/get_exptype_contrasts_assays.pl {wildcards.accession} {input.config_file} {output.metadata}
-        if [ -e "{wildcards.accession}/{wildcards.accession}.condensed-sdrf.tsv" ]; then
-            echo "Trying with condensed..."
+        if [ -e "{params.atlas_exps}/{wildcards.accession}/{wildcards.accession}.condensed-sdrf.tsv" ]; then
+            echo "Trying with condensed in atlas_exps..."
+            organism=$(get_organism_from_condensed_sdrf "{params.atlas_exps}/{wildcards.accession}/{wildcards.accession}.condensed-sdrf.tsv")
+        elif [ -e "{wildcards.accession}/{wildcards.accession}.condensed-sdrf.tsv" ]; then
+            echo "Trying with condensed in atlas_prod..."
```

Comment:
```suggestion
            echo "Trying with condensed in local experiment dir..."
```
I have suffered so much from direct references to atlas directories making our lives difficult in the past that I would avoid reading it as much as possible :-) .